src/playground/ec/.storybook/preview-head.html

```diff
@@ -8,6 +8,11 @@
   integrity="sha512-qTXRIMyZIFb8iQcfjXWCO8+M5Tbc38Qi5WzdPOYZHIlZpzBHG3L3by84BBBOiRGiEb7KKtAOAs5qYdUiZiQNNQ=="
   crossorigin="anonymous"
 ></script>
+<script>
```

Comment:
why this..? Or, i mean, it's true that we were not handling the js fallback in storybook but as long as we don't add the no-js class to the html tag in the storybook iframe we are just adding an unused class "has-js" when the script is run...

Reply:
that's what is defined in our guidelines (https://ec.europa.eu/component-library/ec/getting-started/), so I thought it would be nice to be able to test the no-js behavior when needed

Reply:
ok, but we need to add the no-js class to the html tag in the storybook iframe for this to happen, in this script we are removing a class that we never set, that's what i'm saying..

Reply:
i'll try to expain this better:
Our components have, sometimes, styles based on the no-js class.
What we do in the website, and what we "recommend" the user to do, is to add this class to the html tag of their pages for then replacing that (but it's irrelevant the replacement, it could be simply removed, since we only use the no-js class..) once the dom is ready.

If we want to reproduce this in storybook we would at least need to have the no-js class set by default on the html tag in the storybook iframe, but at the same time storybook would not work with javascript disabled, so in the real world we should provide ourselves a way to only disable the ecl js in storybook for giving the users the chance to test the "no js" behaviour of our components.
(If we had an opt-in approach for the autoInit in our templates this would be easier, it would be enough to add the no-js class on the #root wrapper when the user disables the auto-init, but it's not the case..)

My proposal is to simply remove this, for the moment, it is completely useless, if we want we can think about how to offer this option to the users in the future.